### DISCUSSION TITLE
Color Picker field on change event listener

### DISCUSSION THREAD

**Discussion by Saad5400** [filamentphp/filament#14051] [Help]
### Package

Form builder

### Package Version

v3.2

### How can we help you?

How can I add an event listener to the Color Picker? on change doesn't seem to work when using the color picker, but it works with manual input.

**Comment by leandrocfe** [comment]
You mean, this?

```php
ColorPicker::make('color')
    ->live()
    ->afterStateUpdated(function (string $state) {
        //$state
    })
```

**Reply by Saad5400** [reply]
No sorry I wasn't specific, I meant client side using JavaScript.
